### PR TITLE
Update/readme

### PR DESCRIPTION
**Pull request by ToniRV**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,9 +97,41 @@ If you found any of the above modules useful, we would really appreciate if you
...
+  author = {A. Rosinol, A. Violette, M. Abate, N. Hughes, Y. Chang, J. Shi, A. Gupta, L. Carlone},
+  year = {2021},
+  booktitle = {arxiv},
+  pdf = {https://arxiv.org/pdf/2101.06894.pdf}
+}
+```
+
+## Open-Source Datasets
+
+We use a photo-realistic Unity-based simulator to test Kimera.
```
Comment (lucacarlone) [inline README.md:114]: this seems to hint that Kimera is only tested in simulation, which is not true. Maybe rephrase?

File: README.md [hunk 2]
```diff
@@ -97,9 +97,41 @@ If you found any of the above modules useful, we would really appreciate if you
...
+The simulator provides:
```
Comment (lucacarlone) [inline README.md:115]: the description should belong to the datasets themselves: why listing topics here?

**Review by lucacarlone — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ToniRV) [reply README.md:115]: I might remove the topics, yes, but I would keep the list of info in the dataset to make it easy for the reader to decide if the dataset is of interest. IMHO that is nicer than just providing the link.

File: README.md (hunk 1 above)
Comment (ToniRV) [reply README.md:114]:
```suggestion
In addition to the [real-life tests](http://ci-sparklab.mit.edu:8080/job/MIT-SPARK-Kimera/job/master/VIO_20Euroc_20Performance_20Report/) on the [Euroc](https://projects.asl.ethz.ch/datasets/doku.php?id=kmavvisualinertialdatasets) dataset, we use a photo-realistic Unity-based simulator to test Kimera.
```

File: README.md [hunk 3]
```diff
@@ -97,9 +97,41 @@ If you found any of the above modules useful, we would really appreciate if you
...
+- RGB Stereo camera (/tesse/left_cam, /tesse/right_cam)
+- Depth camera (/tesse/depth_cam)
+- Ground-truth 2D Semantic Segmentation (/tesse/seg_cam)
+- IMU data (/tesse/imu)
+- Ground-Truth Odometry (/tesse/odom)
+- 2D Lidar (/tesse/front_lidar/scan)
+- TF (ground-truth odometry of robots, and agents) (/tf)
+- Static TF (ground-truth poses of static objects) (/tf_static)
```
Comment (ToniRV) [inline README.md:123]:
```suggestion
- RGB Stereo camera
- Depth camera
- Ground-truth 2D Semantic Segmentation
- IMU data
- Ground-Truth Odometry
- 2D Lidar
- TF (ground-truth odometry of robots, and agents)
- Static TF (ground-truth poses of static objects)
```

File: README.md (hunk 2 above)
Comment (lucacarlone) [reply README.md:115]: agreed!

**Review by lucacarlone — APPROVED** [review]
